app/src/main/java/com/example/egida/presentation/viewModel/MainViewModel.kt

```diff
@@ -17,11 +24,14 @@ class MainViewModel : ViewModel() {
     private val userAuthUseCase: UserAuthUseCase by lazy { Dependencies.authUseCase() }
     private val userDbUseCase: UserDbUseCase by lazy { Dependencies.userDbUseCase() }
     private val dayUseCase: DayUseCase by lazy { Dependencies.dayUseCase() }
+    @SuppressLint("StaticFieldLeak")
+    private val activity = Activity()
```

Comment:
OMG.
VM should never reference to activity. Leaks (instead of fix, you suppress them), crashes and incorrect behavior are possible result.